README.md

```diff
@@ -4,6 +4,10 @@ This is an SDK to simplify connecting to the [Smartsheet API](http://www.smartsh
 
 *Please note that this SDK is beta and may change significantly in the future.*
 
+## System Requirements
+
+The SDK supports all versions of Node.js.
```

Comment:
Considering at the time of the release of this SDK the current version of Node.js was 0.10 this isn't too far from being aboslutely accurate. 0.10 is deprecated at this point, so it is probably safe to say that we support back as far as 6.x since that is the active LTS version of Node.js. 5.x doesn't have long term support and 4.x is in maintenance mode at this point. After 4.x is 0.12.x and 0.10.x which are deprecated.